package.json

```diff
@@ -50,11 +50,11 @@
     "precdn": "yarn run build",
     "cdn:deploy": "node ./scripts/deploy.js",
     "cdn": "npm-run-all secrets cdn:deploy",
-    "preopen-styleguide-pr": "rimraf sandbox && yarn run secrets",
-    "open-styleguide-pr": "node ./scripts/open-styleguide-pr",
+    "prenew-version-pr-generator": "rimraf sandbox && yarn run secrets",
+    "new-version-pr-generator": "node ./scripts/new-version-pr-generator",
     "copy-polaris-tokens": "rimraf ./src/styles/polaris-tokens && shx cp -r ./node_modules/@shopify/polaris-tokens/dist ./src/styles/polaris-tokens",
     "prepublish": "in-publish && yarn run build || :",
-    "postpublish": "in-publish && npm-run-all cdn open-styleguide-pr || :",
+    "postpublish": "in-publish && npm-run-all cdn new-version-pr-generator || :",
```

Comment:
Right now we have that `|| :` at the end of this line so that this command never fails with a non-zero exit code, I think we should remove that and just handle the exit code ourselves in the script